django-stubs/contrib/postgres/search.pyi

```diff
@@ -104,9 +104,27 @@ class SearchHeadline(Func):
 class TrigramBase(Func):
     def __init__(self, expression: _Expression, string: str, **extra: Any) -> None: ...
 
+class TrigramWordBase(Func):
+    def __init__(self, expression: _Expression, string: str, **extra: Any) -> None: ...
```

Comment:
Hmm Django documentation has the arguments in the other order, for `TrigramWordSimilarity` https://docs.djangoproject.com/en/4.1/ref/contrib/postgres/search/#django.contrib.postgres.search.TrigramWordSimilarity

```
TrigramWordSimilarity(string, expression, **extra)
```

Or is the documentation wrong?

Reply:
The documentation is correct and I need to fix this, thanks for double checking!

IMO it's super confusing that the arguments are swapped with the other trigram functions, but the rationale is given in the Django PR: https://github.com/django/django/pull/14833#discussion_r702749182